Character.cs

```diff
@@ -8,16 +8,26 @@ namespace ConsoleInitApp
 {
     class Character
     {
-        public string CharacterName { get; set; }
-        //public Action ActionTaken { get; set; }
-        public int HPValue { get; set; }
-        public int DamageValue { get; set; }
+        public string Name { get; set; }
+        public int HP { get; set; }
+        public int Damage { get; set; }
 
-        public Character(string characterName, int hpValue, int damageValue)
+        public Character()
```

Comment:
So here I just created what is often called a NULL constructor or a dummy constructor. This allows us to create an instance of an object without knowing the details ahead of time.